src/plugins/gui/providers/banxaProvider.ts

```diff
@@ -594,7 +594,54 @@ export const banxaProvider: FiatProviderFactory = {
             let insideInterval = false
 
             if (direction === 'buy') {
-              await showUi.openExternalWebView({ url: banxaQuote.data.order.checkout_url })
+              await showUi.openExternalWebView({
+                providerId,
+                url: banxaQuote.data.order.checkout_url,
+                deeplinkHandler: async link => {
+                  if (link.direction !== 'buy') return
+
+                  const orderResponse = await banxaFetch({ method: 'GET', url, hmacUser, path: `api/orders/${banxaQuote.data.order.id}`, apiKey })
+                  const order = asBanxaOrderResponse(orderResponse)
+
+                  switch (link.query.status) {
+                    case 'success': {
+                      console.log('Banxa WebView launch buy success: ' + link.uri)
```

Comment:
We already log tracking events, don't need to mention "buy success."

Remove completely, imo, our logging is so noisy and there's no real value to this considering we log the event immediately below.

To capture logging for the other cases, consider adding new logging events (not as conversions of course) and sync up with Peter. Adding these events would help with analytics on why/if things fail and if people are backing out of their orders for some reason.

Reply:
You didn't add new events, but that can be a separate task. Please create the task and tag Peter and I on it as collaborators.